lib/matplotlib/contour.py

```diff
@@ -1240,11 +1309,10 @@ def _process_linewidths(self):
                     linewidths = linewidths * nreps
                 if len(linewidths) > Nlev:
                     linewidths = linewidths[:Nlev]
-            tlinewidths = [(w,) for w in linewidths]
+            tlinewidths = [w for w in linewidths]
```

Comment:
I think you're guaranteed a list of linewidths from both paths above?
```suggestion
            tlinewidths = linewidths
```